### PR TITLE
Release 0.8.3

### PR DESCRIPTION
**Pull request by sergiuilie**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,25 @@
...
+
+#### Added
+- Syntax highlighting for variables and paragraphs in COBOL file enabled
+
+#### Fixes
+- Java automatic check now recognizes all supported JRE/JDK java implementations
+
 ## [0.8.2]
 
+#### Changes
```
[the remark this answers is not in the document]
Comment (sergiuilie) [reply CHANGELOG.md]: will take notes about that

[the remark this answers is not in the document]
Comment (sergiuilie) [reply CHANGELOG.md]: @DStatWriter please take notes and fix it within the next release

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,25 @@
...
 - Updates to README
 
 ## [0.8.1]
 
-- Automatic check for Java added before launching the LSP server.
+#### Added
+- Automatic check for Java added before launching the LSP server
+
+#### Changed
```
Comment (filipkroupa) [inline CHANGELOG.md]: Changes vs Changed - please fix in future release

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,25 @@
...
+#### Changes
 - Updates to README
 
 ## [0.8.1]
 
-- Automatic check for Java added before launching the LSP server.
+#### Added
+- Automatic check for Java added before launching the LSP server
+
+#### Changed
```
Comment (filipkroupa) [inline clients/cobol-lsp-vscode-extension/CHANGELOG.md]: Changes vs Changed - please fix in future release

File: CHANGELOG.md (hunk 2 above)
Comment (DStatWriter) [reply CHANGELOG.md]: Any objection to me fixing it now in main development branch?

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DStatWriter) [reply CHANGELOG.md]: Changed and pull request made, ready for merging with next defect fix.

**Pull request merged**